CHANGES_NEXT_RELEASE

```diff
@@ -8,3 +8,4 @@ Fix: Changing cygnus port does not work (#1698)
 [cygnus-ngsi] Changed log level for some events (#1769)
 [cygnus-ngsi][PostgisSink, PostGRESQLSink] Support for native Postgres and Postgis types (#1780)
 [cygnus-ngsi][MySQL] Support for native MySQL types (#1780)
+[cygnus-ngsi][HDFS] Encode value when it cotains commas (#1785)
```

Comment:
Typo

```suggestion
[cygnus-ngsi][HDFS] Encode value when it contains commas (#1785)
```

Reply:
Thks. Fixed in [0a63bae](https://github.com/telefonicaid/fiware-cygnus/pull/1799/commits/0a63bae73f14a66383707ae6c1ac6111625ad34f)